src/language-handlebars/utils.js

```diff
@@ -1,16 +1,80 @@
 "use strict";
 
+function isUppercase(string) {
+  return string.toUpperCase() === string;
+}
+
 function isGlimmerComponent(node) {
-  if (node.type !== "ElementNode") {
-    return false;
-  }
+  return (
+    isNodeOfSomeType(node, ["ElementNode"]) &&
+    typeof node.tag === "string" &&
+    (isUppercase(node.tag[0]) || node.tag.includes("."))
+  );
+}
+
+function isWhitespaceNode(node) {
+  return isNodeOfSomeType(node, ["TextNode"]) && !/\S/.test(node.chars);
```

Comment:
```suggestion
  return isNodeOfSomeType(node, ["TextNode"]) && /\s/.test(node.chars);
```

Reply:
Seems don't need negative, right?

Reply:
Wait, not equal, if all `\s`

Reply:
Sorry, I think my suggestion is not right.

Reply:
```js
!/\S/.test('')
true
/\s/.test('')
false
```

Reply:
/^\s*$/ should be equal, but old one better

Reply:
Removed the last commit 😉